st2common/tests/unit/test_action_alias_utils.py

```diff
@@ -14,293 +14,64 @@
-        _, value, _ = STR_SINGLE_QUOTE_PARSER.parse(start, stream)
-        self.assertEqual(value, 'double spaced value2')
-
-        start = len(stream) - 2
-        self.assertFalse(STR_SINGLE_QUOTE_PARSER.is_applicable(stream[start]),
-                         'Should not be parsable.')
-
-    def testIncompleteStringParsing(self):
-        stream = "some meaningful 'spaced ."
-
-        start = len('some meaningful ')
-        self.assertTrue(STR_SINGLE_QUOTE_PARSER.is_applicable(stream[start]), 'Should be parsable.')
-        try:
-            STR_SINGLE_QUOTE_PARSER.parse(start, stream)
-            self.assertTrue(False, 'Parsing failure expected.')
-        except content.ParseException:
-            self.assertTrue(True)
-
-
-class TestJsonValueParser(TestCase):
-
-    def testJsonParsing(self):
-        stream = 'some meaningful {"a": "b"} something else skippable {"c": "d"} end.'
-
-        start = len('some meaningful ')
-        self.assertTrue(JsonValueParser.is_applicable(stream[start]), 'Should be parsable.')
-        _, value, _ = JsonValueParser.parse(start, stream)
-        self.assertEqual(value, '{"a": "b"}')
-
-        start = len('some meaningful {"a": "b"} something else skippable ')
-        self.assertTrue(JsonValueParser.is_applicable(stream[start]), 'Should be parsable.')
-        _, value, _ = JsonValueParser.parse(start, stream)
-        self.assertEqual(value, '{"c": "d"}')
-
-        start = len(stream) - 2
-        self.assertFalse(JsonValueParser.is_applicable(stream[start]), 'Should not be parsable.')
-
-    def testEndJsonParsing(self):
-        stream = 'some meaningful {"a": "b"}'
-
-        start = len('some meaningful ')
-        self.assertTrue(JsonValueParser.is_applicable(stream[start]), 'Should be parsable.')
-        _, value, _ = JsonValueParser.parse(start, stream)
-        self.assertEqual(value, '{"a": "b"}')
-
-    def testComplexJsonParsing(self):
-        stream = 'some meaningful {"a": "b", "c": "d", "e": {"f": "g"}, "h": [1, 2]}'
-
-        start = len('some meaningful ')
-        self.assertTrue(JsonValueParser.is_applicable(stream[start]), 'Should be parsable.')
-        _, value, _ = JsonValueParser.parse(start, stream)
-        self.assertEqual(value, '{"a": "b", "c": "d", "e": {"f": "g"}, "h": [1, 2]}')
-
-    def testIncompleteStringParsing(self):
-        stream = 'some meaningful {"a":'
-
-        start = len('some meaningful ')
-        self.assertTrue(JsonValueParser.is_applicable(stream[start]), 'Should be parsable.')
-        try:
-            JsonValueParser.parse(start, stream)
-            self.assertTrue(False, 'Parsing failure expected.')
-        except content.ParseException:
-            self.assertTrue(True)
+from st2common.models.utils.action_alias_utils import ActionAliasFormatParser
 
 
 class TestActionAliasParser(TestCase):
-    def test_default_key_value_param_parsing(self):
-        # Empty string
+    def testEmptyString(self):
```

Comment:
Sure. The original testing suite had both camelCase and underscores, so I just went with the former. Will change. :)